ci/images/ci-runner/hack/bin/run-plnsvc-test.sh

```diff
@@ -6,7 +6,7 @@ set -x
 
 echo "Run pipeline-service tests..."
 if kubectl -n default exec pod/ci-runner -- \
-    sh -c "/workspace/sidecar/bin/plnsvc_test.sh"; then
+    sh -c "/workspace/sidecar/bin/plnsvc_test.sh --debug"; then
```

Comment:
since you did not touch plnsvc_test.sh, I'm assuming that it was already looking for `--debug` and employ `set -x` of some equivalent ?

Reply:
That's an error, thanks for catching it. I was supposed to update `ci/images/ci-runner/hack/sidecar/bin/plnsvc_test.sh`